crates/interfaces/src/p2p/bodies/downloader.rs

```diff
@@ -1,6 +1,25 @@
 use crate::p2p::downloader::{DownloadStream, Downloader};
 use reth_primitives::{BlockLocked, SealedHeader};
 
+/// The block response
+#[derive(PartialEq, Eq, Debug)]
+pub enum BlockResponse {
+    /// Full block response (no transactions and ommers)
```

Comment:
WDYM no transactions? Is there an extra "no" here?

 https://github.com/paradigmxyz/reth/blob/617ae3604d94ff52f5d73e624ba03db669038c19/crates/primitives/src/block.rs#L31

Reply:
fixed